package.json

```diff
@@ -48,6 +48,7 @@
     "@metamask/eslint-config-jest": "^12.1.0",
     "@metamask/eslint-config-nodejs": "^12.1.0",
     "@metamask/eslint-config-typescript": "^12.1.0",
+    "@metamask/gas-fee-controller": "^21.0.0",
```

Comment:
This has been added because GasFeeController is now a peerDependency in TransactionController